backend/tracim_backend/tests/functional/test_system.py

```diff
@@ -279,6 +279,20 @@ def test_api__get_config__ok_200__nominal_case(self, patched_get, web_testapp):
         assert supported_file_types[2]["associated_action"] == "edit"
 
 
+@pytest.mark.usefixtures("base_fixture")
+@pytest.mark.parametrize("config_section", [{"name": "call_jitsi_test"}], indirect=True)
+class TestConfigCallJitsi(object):
+    """
+    Tests for /api/system/config
+    """
+
+    def test_api__get_config__ok_200__nominal_case(self, web_testapp):
+        web_testapp.authorization = ("Basic", ("admin@admin.admin", "admin@admin.admin"))
+        res = web_testapp.get("/api/system/config", status=200)
+        assert res.json_body["call__enabled"] is True
+        assert res.json_body["call__unanswered_timeout"] == 20
```

Comment:
same question here about timeout

Reply:
The value have to be the same as the one in the file tests_configs.ini

Reply:
ok so maybe you can decrease value to not slow the test (its an idea)

Reply:
ok